FirebaseAdmin/FirebaseAdmin.Tests/Auth/FirebaseAuthTest.cs

```diff
@@ -136,7 +136,7 @@ public async Task VerifyIdTokenCancel()
             var canceller = new CancellationTokenSource();
             canceller.Cancel();
             var idToken = await FirebaseTokenVerifierTest.CreateTestTokenAsync();
-            await Assert.ThrowsAsync<OperationCanceledException>(
+            await Assert.ThrowsAnyAsync<OperationCanceledException>(
```

Comment:
Why was this change necessary? Can we revert it?

Reply:
As we're now passing the CancellationToken to the Semaphore, this will throw a TaskCanceledException if the given CancellationToken has been cancelled (which is a more derived type of OperationCanceledException https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.taskcanceledexception)